discord/colour.py

```diff
@@ -226,5 +226,15 @@ def blurple(cls):
     def greyple(cls):
         """A factory method that returns a :class:`Colour` with a value of ``0x99aab5``."""
         return cls(0x99aab5)
+    
+    @classmethod
+    def darker_greyple(cls):
```

Comment:
This was brought up a few days ago again in the discord.py server and the idea generally doesn't seem too bad. Incidentally, a better name was mentioned there so I'll propose it here.

```suggestion
    def dark_theme(cls):
```

The documentation can then be updated to refer to the fact that this is the dark theme's background colour and the appropriate versionadded string should you want it.